notebooks/00_quick_start/wide_deep_model_movielens.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>I'm curious, why MAE and not RMSE? is what they recommend in the paper?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
No, It is just an example of how to printing out additional metrics via TensorFlow's native functions.

Btw, model's default `average_loss` is Mean Square Error, and thus just `sqrt(average_loss)` is the RMSE--tested w/ our `rmse` function too. That's why I didn't use RMSE and showed MAE instead as an 'additional' metrics example. Nothing deep meaning here haha. Will add more context to the cell.